bower.json

```diff
@@ -1,39 +1,31 @@
 {
-  "name": "cedar",
-  "version": "0.0.0",
+  "name": "arcgis-cedar",
   "homepage": "https://github.com/esri/cedar",
   "authors": [
     "Dave Bouwman <dave@davebouwman.com>"
   ],
   "description": "Charting for the ArcGIS platform",
   "main": [
-    "src/cedar.js",
-    "src/charts/bar-horizontal.json",
-    "src/charts/bar.json",
-    "src/charts/bubble.json",
-    "src/charts/pie.json",
-    "src/charts/scatter.json",
-    "src/charts/sparkline.json",
-    "src/charts/time.json"
+    "dist/cedar.js",
+    "dist/charts/bar-horizontal.json",
+    "dist/charts/bar.json",
+    "dist/charts/bubble.json",
+    "dist/charts/pie.json",
+    "dist/charts/scatter.json",
+    "dist/charts/sparkline.json",
+    "dist/charts/time.json"
   ],
   "license": "Apache 2",
   "private": true,
```

Comment:
I think this needs to be removed or false, otherwise you won't be able to publish to bower.